task_manager/urls.py

```diff
@@ -0,0 +1,98 @@
+from django.urls import path
+
+from task_manager.views import (
+    index,
+    TaskTypeListView,
+    PositionListView,
+    TeamListView,
+    WorkerListView,
+    ProjectListView,
+    TaskListView,
+    TaskTypeDetailView,
+    PositionDetailView,
+    TaskDetailView,
+    ProjectDetailView,
+    WorkerDetailView,
+    TeamDetailView,
+    TaskTypeCreateView,
+    PositionCreateView,
+    TeamCreateView,
+    WorkerCreateView,
+    ProjectCreateView,
+    TaskCreateView,
+    TaskTypeUpdateView,
+    PositionUpdateView,
+    TeamUpdateView,
+    WorkerUpdateView,
+    ProjectUpdateView,
+    TaskUpdateView,
+    TaskTypeDeleteView,
+    PositionDeleteView,
+    TeamDeleteView,
+    WorkerDeleteView,
+    ProjectDeleteView,
+    TaskDeleteView,
+    task_set_completed,
+    task_set_not_completed,
+)
```

Comment:
W przypadku takich kolosów lepiej jest:
```suggestion
from task_manager import views
```
i wtedy odnosić się po
```
views.TaskTypeListView
```